kaggle_environments/main.py

```diff
@@ -230,21 +303,26 @@ def http_request(request):
             del params[key]
 
     body = request.get_json(silent=True, force=True) or {}
-    req = parse_args({**params, **body})
-    resp = action_handler(req)
+    args = {**params, **body}
+    if "render" in args:
```

Comment:
This allows you to pass a render parameter in when running in http-server mode so you can do something like
`kaggle-environments http-server --port 8080` then visit `127.0.0.1:8080?environment=halite&agents[]=random&agents[]=./path/to/my/agent&render={"mode":"html"}` to instantly render an episode (refreshing the page generates and renders a new episode each time).

Reply:
Or with the new display syntax: `127.0.0.1:8080?environment=halite&agents[]=random&agents[]=./path/to/my/agent&display=html`